kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/kommunicate/utils/KmHelper.java

```diff
@@ -70,7 +75,7 @@ public void onSuccess(Object message) {
                 @Override
                 public void onFailure(Object error) {
                     dialog.dismiss();
-                    Toast.makeText(context, context.getString(R.string.unable_to_create_conversation) + " : " + error, Toast.LENGTH_SHORT).show();
+                    Toast.makeText(context, context.getString(io.kommunicate.R.string.unable_to_create_conversation) + " : " + error, Toast.LENGTH_SHORT).show();
```

Comment:
@reytum is io.kommunicate.R intentional here?

Reply:
Done